src/arch/helperadvsimd.h

```diff
@@ -43,6 +43,14 @@ typedef int32x2_t vint;
 static INLINE int vavailability_i(int name) { return 3; }
 static INLINE void vprefetch_v_p(const void *ptr) { }
 
+static INLINE int vtestallones_i_vo32(vopmask g) {
+  return vget_lane_u32(vpmin_u32(vand_u32(vget_low_u32(g), vget_high_u32(g)), vand_u32(vget_low_u32(g), vget_high_u32(g))), 0);
```

Comment:
I knew I should have compiled it before sending it to you! :)

Here is another implementation:

```
uint32x2_t x0 = vand_u32(vget_low_u32(x0), vget_high_u32(0));
uint32x2_t x1 = vpmin_u32(x0, x0);
return vget_lane_u32(x1,0);
```
Disclaimer: I haven't tested this one too! But it seems to make sense to me, in the hypothesis that the `uint32_t` lanes are either `0x0` or `0xffffffff`.